index.js

```diff
@@ -1,7 +1,13 @@
 var path = require("path");
+var bourbon = require("bourbon");
+var neat = require("bourbon-neat");
+
+var includePaths = [];
+
+includePaths.concat(path.join(__dirname, "core"));
+includePaths.contact(bourbon.includePaths);
```

Comment:
no "contact" :)

Reply:
HAHAHA!!! I was trying to `contact` you.

You had something like this in mind, correct?